packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.types.ts

```diff
@@ -145,6 +146,11 @@ export type IDropdownStyleProps = Pick<IDropdownProps, 'theme' | 'className' | '
    * This is primarily provided for backwards compatibility.
    */
   calloutClassName?: string;
+
+  /**
+   * Prop to notify on what edge the dropdown callout was positioned respective to the title.
+   */
+  calloutRenderEdge?: RectangleEdge;

```

Comment:
@dzearing I noticed this [FOLDER](https://github.com/OfficeDev/office-ui-fabric-react/tree/master/packages/office-ui-fabric-react/src/utilities) from where I exported the `RectangleEdge` interface does not have an `index` file exporting all this utilities even though  every folder inside of it has it. Was the intention here to keep this utilities private for internal usage?

Should we better make all this utilities exposed in [HERE](https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/office-ui-fabric-react/src/Utilities.ts) along with the `@uifabric/utilities` package instead of having it exported from Dropdown? So that the import would be `import {RectangleEdge} from 'office-ui-fabric-react/lib/Utilities`.